### DISCUSSION TITLE
`vimcmd_symbol` for nushell?

### DISCUSSION THREAD

**Discussion by TornaxO7** [starship/starship#5539] [Q&A] [answered]
Hi! May I ask if I can get `vimcmd_symbol` running for `nushell`?

**Comment by davidkna** [comment, marked as answer]
Unlike other shells nushell won't re-render the prompt on mode-changes, unless we work out a way around that. Alternatively you can use the native nushell [`PROMPT_INDICATOR_*` feature.](https://www.nushell.sh/book/coloring_and_theming.html#prompt-configuration-and-coloring)

**Reply by TornaxO7** [reply]
Oh ok. Thank you for the information! :)
